### DISCUSSION TITLE
jupyter book and readthedocs

### DISCUSSION THREAD

**Discussion by parmentelat** [executablebooks/meta#294] [Q&A]
I'd be very interested in being able to host a jupyter-book under readthedocs as a CD artefact, like other sphinx-powered projects
A couple months back it seemed like that was a cool idea but not too close to happen, but just now I ran into
https://myst-nb.readthedocs.io/en/latest/

and so I am excited to ask again about the general availability of this ?
would be soooo cool !

thanks in any case for all the good work, so helpful !

**Comment by choldgraf** [comment]
To quickly clarify - most of Jupyter Book is just an opinionated _distribution of Sphinx_ - almost all of the underlying features of Jupyter Book are Sphinx extensions. Jupyter Book is not natively hosted on Sphinx (though there is definitely hope to do this (https://github.com/executablebooks/jupyter-book/issues/1091). However, you can re-create much of the "Jupyter Book experience" by using your own custom Sphinx configuration.

**Comment by parmentelat** [comment]
ok, thanks, but can you please just elaborate on how it was possible at all to get the myst-nb doc hosted under readthedocs ?

**Reply by choldgraf** [reply]
Have you used Sphinx before? That is the way to do it, you just add myst_nb to your list of extensions. perhaps we should write up a short tutorial or something...

**Comment by parmentelat** [comment]
yes, I do know about sphinx; the way both tools look like for a user like me are very different (in particular with jb I write `_toc.yml` and `_config.yml`, while with sphinx it's `conf.py` and `index.rst`)
so if you ask me a short tutorial on the general picture would be helpful

because if jb runs sphinx behind the scenes, it must already produce all the artefacts (again stuff like `conf.py` and `index.rst`), is there an option to stop the workflow at that point - in order to produce / maintain those artefacts so as to publish on rtd ? unless sphinx can already do its job from `_toc.yml` and `_config.yml` ?

that's where a clarification would help the most as far as I am concerned :)

-- thansk !

**Reply by chrisjsewell** [reply]
> unless sphinx can already do its job from _toc.yml and _config.yml ?

It probably will soon, but not just yet
